### DISCUSSION TITLE
Full content not getting displayed in the content player

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Ed/Community#475] [Bugs] [answered]
Hello All,

I'm currently working on the SunbirdEd 4.6 version.

**Content Type -** Question Set
I've created a MCQ in a Question set. The MCQ is having a long passage as a question 
![image](https://user-images.githubusercontent.com/107101132/234175884-03c39e48-acb9-4ec0-8713-a5f582431323.png)

When I'm trying to play this Question set in the QUML player, my passage is getting cropped/ not able to see the whole content of passage.
![image](https://user-images.githubusercontent.com/107101132/234176006-8e58bef0-1d51-4a96-94d6-495214ded911.png)

**What can be done to display the whole passage in the content player as client is facing this issue?**

cc: @rajeevsathish @Krishnaj20 @vinukumar-vs

**Comment by Abhign** [comment]
@rajeevsathish @vinukumar-vs Can you check this ticket.

CC: @Krishnaj20

**Reply by vinukumar-vs** [reply]
@ashokreddy1208 Do we have any scenario tested for this? 
@sajeshKayyath Any suggestion?

@nidhikachhawa24 I would like to move this discussion to Inquiry building block.

@Krishnaj20 @sajeshkayyath @vaibhavbhuva

**Reply by sajeshkayyath** [reply]
@nidhikachhawa24 - we checked this with the latest version (5.5.0) of QuML player and not noticing any issue. we will keep you posted if there was any issue in the prior version

@Krishnaj20 @vinukumar-vs  @vaibhavbhuva

**Comment by ashokreddy1208** [comment]
@nidhikachhawa24 @vinukumar-vs This issue is resolved in 5.1 Inquiry, Please find issue id here for your reference - [IQ-64](https://project-sunbird.atlassian.net/browse/IQ-64), [IQ-68](https://project-sunbird.atlassian.net/browse/IQ-68).

 We tested on Sunbird ED 5.1 and 5.2 Versions, we are unable to replicate this issue. 
<img width="1383" alt="image" src="https://user-images.githubusercontent.com/58033030/234300812-98b7933f-5c58-4846-88c7-30513fe95a88.png">

@Krishnaj20 @vinukumar-vs @vaibhavbhuva

**Reply by Krishnaj20** [reply]
thanks @ashokreddy1208 @sajeshkayyath .

@sajeshkayyath - Pls share the patch applied in 5.1 inquiry to fix this issue in this thread so that @nidhikachhawa24 can apply it in her instance.

**Reply by vaibhavbhuva** [reply, marked as answer]
@nidhikachhawa24 
Please find the PR link below which contains the patch for the same: 
https://github.com/project-sunbird/sunbird-quml-player/pull/314/files

cc: @sajeshkayyath @Krishnaj20

**Reply by Krishnaj20** [reply]
@nidhikachhawa24 - Hope your query is now resolved. Please mark the thread as answered.
